index.html

```diff
@@ -526,6 +526,21 @@ <h3>
           can be <a>installed</a> is left to implementers.
         </p>
       </section>
+      <section>
+        <h3>
+          Uninstallation
+        </h3>
+        <p>
+          User agents SHOULD provide a mechanism for the user to remove the
+          installed app from the system.
```

Comment:
"from the system" might be overly specific... a UA can "install" a web app to, for example, the new tab page.

Reply:
Maybe just drop that part then... apart from that lgtm

Reply:
Done.